Resources/Private/Fusion/Prototypes/TrackingCode.GA.fusion

```diff
@@ -5,6 +5,9 @@ prototype(Neos.GoogleAnalytics:TrackingCode.GA) < prototype(Neos.GoogleAnalytics
         hasTrackingId = ${!String.isBlank(this.trackingId)}
         hasNoContainerId = ${String.isBlank(this.configuration.tagManager.id)}
     }
+    
+    anonymizeString = ', {"anonymize_ip": true}'
```

Comment:
I think it would be better to put the comma when you concatenate the strings. 
Also the parameters should be a `Neos.Fusion:DataStructure` which can be JSON encoded.
This way they can then be easily extended.

Reply:
I stuck rendering the parameters setting into the Neos.Fusion:DataStructure. Can you give me a hint how to achieve that in fusion? When defining a new Neos.Fusion:DataStructure every element gets a new [key] but I want it to use the key defined in the configuration already.

Reply:
Thank you, it is done!